client/app/components/stacked-bars/component.js

```diff
@@ -0,0 +1,62 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+    chartSelector: '.widget-stackedBar',
+    data:[],
+    pattern: ['#1f77b4', '#aec7e8', '#ff7f0e', '#ffbb78', '#2ca02c', '#98df8a', '#d62728', '#ff9896', '#9467bd', '#c5b0d5', '#8c564b', '#c49c94', '#e377c2', '#f7b6d2', '#7f7f7f', '#c7c7c7', '#bcbd22', '#dbdb8d', '#17becf', '#9edae5'],
+    calculateTotal(items){
+        let total = 0;
+        for (var value of items) {
+          total += value.doc_count;
+        }
+        return total;
```

Comment:
that's interesting, it probably is, but this will guarantee a much more robust and hard to break total.